### PR TITLE
Fixes scroll when viewing Markdown files

### LINKED ISSUE #651: can not scroll the view in MarkDown file
I cant scroll the view in the  markdown file

### LINKED ISSUE #654: Readme cannot scroll
Cannot scroll when readme is long

### PR DESCRIPTION
**Pull request by bibstha**
Fixes #654 #651 #650 #632 #613

Due to the `overflow: hidden`, when you open a Readme.md or any other markdown file on iOS, it scrolling is completely disabled.

With this, you should be able to scroll again!

Screenshot: https://drive.google.com/open?id=1y4OAwe2qyTG1ATHT7upZiZTnjjxFHNXx
cc: @thedillonb

### CODE REVIEW

**Comment by spacebahr** [comment]
Thanks I have been trying see what issues other people are having with the
ras pi

**Comment by Faroop** [comment]
Nope still doesn’t allow for scrolling.

**Comment by mocarvalhomoc** [comment]
Replay what??

Marcelo Carvalho
Em 21 de abr de 2020 02:25 -0300, Faraz <notifications@github.com> escreveu:

**Comment by mocarvalhomoc** [comment]
He manipulated people

**Comment by bibstha** [comment]
Did you try compiling the changes and loading it on a simulator or your iPhone? 
Please don’t confuse this with the released app version on App Store. I’m waiting for the maintainer to test and merge and then release the patch.

**Comment by fasm** [comment]
Good luck. This app hasn’t received an update in a year. The author/maintainer of the App Store app is MIA.

**Review by borghetti19 — APPROVED** [review]

**Comment by thedillonb** [comment]
Outstanding work @bibstha! I will work to produce a new image for the app store. Thank you for your contribution!

**Comment by bibstha** [comment]
Hi @thedillonb any chance of a new release soon? I use it to read and write my notes and this fix will make the experience much better.

**Comment by thedillonb** [comment]
@bibstha yes. It's actually in progress right now. There was an issue with the GitHub account I  traditionally give to Apple to use in reviewing the application.

**Comment by jpinkham** [comment]
I can confirm that the scrolling issue is now resolved! Download the latest version from the App Store (with a nice shoutout to @bibstha in release description), v2.19.1, which was released about a week ago. Thank you thank you thank you 😍

**Pull request merged**
